### PR TITLE
KYC Tweaks 2

### PR DESCRIPTION
**Pull request by cballantyne**

### CODE REVIEW

File: web/web/templates/web/index_not_verified.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            <p>
+                We’re excited to have you join the community.
+            </p>
+            <p>
+                You can use the platform to discover, invest and contribute to interesting ICO projects or create your own.  
+            </p>
+            <p>
+                Take some time to explore the different features.
+            </p>
+            <p>
```
Comment (ejjick) [inline web/web/templates/web/index_not_verified.html]: IMHO, at some point, these three paragraphs should just be an include file triggered by an if-statement.

File: web/web/views.py [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
 def index(request):
-    return render(request, 'web/index.html')
+    # check if user is verified
+    profile_verified = False
+    if Profile.objects.filter(user_id=request.user.id).exists():
+        profile = Profile.objects.get(user_id=request.user.id)
+        if profile.verified:
+            profile_verified = True
+
+    if profile_verified:
```
Comment (ejjick) [inline web/web/views.py]: Replace with `return render(request, 'web/index.html', {profile_verified: profile_verified})` and slight modification to `index.html`?

Comment (cballantyne) [reply web/web/views.py]: Agree with this. But I wasn't sure what the content would be at this stage. I only guessed the content might be quite different and have dashboard type stuff later. That was my only reasoning for doing it this way with separate html file.

File: web/web/templates/web/index_not_verified.html (hunk 1 above)
Comment (cballantyne) [reply web/web/templates/web/index_not_verified.html]: Yeah totally. All depends on the content in the two index pages and how much they differ.

**Pull request merged**